src/views/about/SupportUs.vue

```diff
@@ -2,27 +2,81 @@
   <Section>
     <h2 id="support">Support Us</h2>
     <p>
-      The tools created by the BioThings collaboration are funded by grants
-      [link to biothings-global page that lists grant sources?]. Although the
+      The tools created by the BioThings collaboration are funded by grants from
+      <a href="https://foundation.org/">EXAMPLE FOUNDATION</a>,
+      <a href="https://foundation.org/">EXAMPLE FOUNDATION</a>, and
+      <a href="https://foundation.org/">EXAMPLE FOUNDATION</a>. Although the
       individual data sources included in these tools may be subject to
       restrictions, the tools themselves are free and we would like them to
       remain that way. If you found our tools useful, please help support us by
       spreading the word and demonstrating its value:
     </p>
-    <p>
-      Twitter link, GitHub repo links, link to biothings.io branding page, link
-      to “cite” section below
-    </p>
+    <Center class="buttons">
+      <a
+        href="https://twitter.com/mygenesetinfo?ref_src=twsrc%5Etfw"
```

Comment:
Maybe it would be good to add a twitter icon here, to differentiate from the GitHub links.

Reply:
There is one:

![image](https://user-images.githubusercontent.com/8326331/110171994-86996e00-7dca-11eb-9ca1-f8da63b043a6.png)